python/postprocessing/examples/mhtjuProducerCpp.py

```diff
@@ -0,0 +1,52 @@
+import ROOT
+import os
+ROOT.PyConfig.IgnoreCommandLineOptions = True
+
+from PhysicsTools.NanoAODTools.postprocessing.framework.datamodel import Collection
+from PhysicsTools.NanoAODTools.postprocessing.framework.eventloop import Module
+
+class mhtjuProducerCpp(Module): # MHT producer, unclean jets only (no lepton overlap cleaning, no jet selection)
+    def __init__(self):
+        if "/mhtjuProducerCppWorker_cc.so" not in ROOT.gSystem.GetLibraries():
+            print "Load C++ mhtjuProducerCppWorker worker module"
+            base = os.getenv("NANOAODTOOLS_BASE")
+            if not base:
+                base = "%s/src/PhysicsTools/NanoAODTools"%os.getenv("CMSSW_BASE")
+            ROOT.gROOT.ProcessLine(".L %s/src/mhtjuProducerCppWorker.cc+O"%base)
```

Comment:
if the code is compiled beforehand, one should load the library instead of loading the .cc